src/components/transition/stories/OffsetBehavior.jsx

```diff
@@ -47,6 +47,11 @@ const contentCounterSlideEffect = {
     transform: {scaleY: 1, duration: 'gentle1', origin: 'left top'},
     easing: 'entry',
   },
+  exit: {
+    opacity: 0,
+    duration: 'quick2',
+    easing: 'exit',
+  },
```

Comment:
This problem with missing the `exit` phase wasn't visible with an instant animation bug.